apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/AppController.java

```diff
@@ -106,7 +106,7 @@ public AppController(
   @GetMapping("/search/by-appid-or-name")
   public PageDTO<App> searchByAppIdOrAppName(@RequestParam(value = "query", required = false) String query,
       Pageable pageable) {
-    if (StringUtils.isEmpty(query)) {
+    if (!StringUtils.hasLength(query)) {
```

Comment:
How about we use `Strings.isNullOrEmpty` instead of `!StringUtils.hasLength` which doesn't need the negate operator?
```suggestion
    if (Strings.isNullOrEmpty(query)) {
```

Reply:
Ok, I will fix it.